main.go

```diff
@@ -12,6 +24,101 @@ func init() {
 	_ = flag.Set("logtostderr", "true")
 }
 
+func GetCurrentNode(clientset kubernetes.Clientset) (*corev1.Node, error) {
+	nodeName := os.Getenv("NODE_NAME")
+	if nodeName == "" {
+		return nil, errors.New("Environment variable POD_NAME is not set")
```

Comment:
nit: error message should say `NODE_NAME`

Reply:
Thanks, I just fixed it!